image_services/egis/make_egis_services.py

```diff
@@ -280,12 +339,16 @@ def update_csv(csv_file: str, rasters: List[str]):
         where_clause=f"Name = '{overview_name}'",
     )
 
+    calculate_fields(selection)
     arcpy.management.CalculateFields(
         in_table=selection,
         fields=[
             ['MinPS', '900'],
+            ['MaxPS', '144000'],
```

Comment:
Does this conflict with the `'MaxPS': '910'` fields in `get_raster_metadata`?

Reply:
MinPS/MaxPS control the zoom levels at which a particular item in the mosaic is visible. We set min=0, max=910 for the rasters so they're what shows up when you're zoomed in, and min=900 max=14400 for the overview so it's what shows when you're zoomed out.

This `CaclulateFields` call is operating on `selection`, so it only sets these field values for the overview record, not the individual geotiff records.

Reply:
Got it, thanks.